src/main/resources/avro/bdg.avdl

```diff
@@ -404,6 +404,23 @@ record Variant {
   union { null, string } alternateAllele = null;
 
   /**
+   True if filters were applied for this variant. VCF column 7 "FILTER" any value other
+   than the missing value.
+   */
+  union { null, boolean } filtersApplied = null;
```

Comment:
Right, see commit https://github.com/bigdatagenomics/bdg-formats/pull/108/commits/cd69f8d86db26a8f11059a5c32933c96a2eec68e in #108 for the equivalent `Genotype` changes